src/main/java/com/apigee/edge/config/rest/RestUtil.java

```diff
@@ -138,18 +138,32 @@ public static HttpResponse updateEnvConfig(ServerProfile profile,
     }
 
     public static HttpResponse deleteEnvConfig(ServerProfile profile, 
+									            String resource,
+									            String resourceId)
+	throws IOException {
+    	return deleteEnvConfig(profile, resource, resourceId, null);
+    }
+    
+    public static HttpResponse deleteEnvConfig(ServerProfile profile, 
                                                 String resource,
-                                                String resourceId)
+                                                String resourceId,
+                                                String payload)
```

Comment:
deleteEnvConfig is used by other Mojos as well. Will this additional parameter break others.

Reply:
@msadasivam - there is another method 
`deleteEnvConfig(ServerProfile profile,  String resource, String resourceId)` above this method (overloaded) which should call the new method. Existing methods should call this and work as usual.. I have tested those as well